models/class.js

```diff
@@ -21,8 +21,14 @@ const classSchema = Schema({
   price: {
     type: Number
   },
-  state: {
-    type: Number
+  duration: {
+    type: String
+  },
+  curriculum: {
```

Comment:
this curriculum should be a type of mongoose objected and it should reference the curriculum collection

Reply:
That's right changed